core/persistence-jpa/src/test/java/org/apache/syncope/core/persistence/jpa/inner/AnySearchTest.java

```diff
@@ -175,6 +210,28 @@ public void searchWithNotCondition() {
     }
 
     @Test
+    public void searchCaseInsensitiveWithNotCondition() {
```

Comment:
This test should be replicated in [SearchITCase](https://github.com/apache/syncope/blob/2_0_X/fit/core-reference/src/test/java/org/apache/syncope/fit/core/SearchITCase.java) (like other test cases in this class)